flexget/components/estimate_release/estimate_release.py

```diff
@@ -21,9 +21,13 @@ def estimate(self, entry):
         """
 
         logger.debug(entry['title'])
-        estimators = [
-            e.instance.estimate for e in plugin.get_plugins(interface='estimate_release')
-        ]
+        # If the entry is in a task context, use the task interface to get estimator plugins, so that we don't get ones
+        # that are disabled on that task.
```

Comment:
not sure i follow on how an entry can NOT be in a task context

Reply:
I'm not sure if it really can either. But I remember weirdness when accessing entry.task at some point before, so this was sorta protecting against that. I'm also not really a fan of entry.task, I'm wondering if we shouldn't just switch to passing task as a separate parameter.

Reply:
makes sense. It'll also create a single place for this logic which would be better